packages/compass-collection/src/components/mock-data-generator-modal/script-screen.tsx

```diff
@@ -224,8 +229,7 @@ const mapStateToProps = (state: CollectionState) => {
       schemaAnalysis?.status === SCHEMA_ANALYSIS_STATE_COMPLETE
         ? schemaAnalysis.arrayLengthMap
         : {},
-    // TODO(CLOUDP-333856): When document count step is implemented, get documentCount from state
-    documentCount: 100,
+    documentCount: mockDataGenerator.documentCount,
```

Comment:
Glad this was caught! I should've made sure this was included in the JIRA ticket. 😬